helm/robusta/templates/runner.yaml

```diff
@@ -29,6 +29,14 @@ spec:
             value: "true"
           - name: ENABLE_COLORED_LOGS
             value: "true"
+          - name: PROMETHEUS_ENABLED
+            value: {{ .Values.enablePrometheusStack | quote}}
+          - name: CLUSTER_ID_HASH
```

Comment:
cluster_name and account_id are part of global config, and can change without restart.
We already have it in the runner, might be better to take it from there and hash it